index.md

```diff
@@ -0,0 +1,90 @@
+  </li>
+  <li>
+    <p><strong>Open and debuggable</strong>: WebAssembly is designed to be pretty-printed in a <a href="https://github.com/WebAssembly/design/blob/master/TextFormat.md">textual format</a> for debugging, testing, experimenting, optimizing, learning, teaching, and writing programs by hand. The textual format will be used when <a href="https://github.com/WebAssembly/design/blob/master/FAQ.md#will-webassembly-support-view-source-on-the-web">viewing the source</a> of wasm modules on
+      the web.</p>
+  </li>
+  <li>
+    <p><strong>Part of the open web platform</strong>: WebAssembly is designed to maintain the versionless, feature-tested, and backwards-compatible <a href="https://github.com/WebAssembly/design/blob/master/Web.md">nature of the web</a>. WebAssembly modules will be able to call into and out of the JavaScript context and access browser functionality through the same Web APIs accessible from JavaScript. WebAssembly also supports <a href="https://github.com/WebAssembly/design/blob/master/NonWeb.md">non-web</a> embeddings.</p>
+  </li>
+</ul>
+<p>Read the project's <a href="https://github.com/WebAssembly/design/blob/master/HighLevelGoals.md">high-level goals</a> and consult the <a href="https://github.com/WebAssembly/design/blob/master/FAQ.md">FAQ</a> section for more information.</p>
+<h2>Roadmap</h2>
+<table>
+  <thead>
+    <tr>
+      <th class="text-left"><a href="https://github.com/WebAssembly/design/blob/master/MVP.md">MVP</a></th>
+      <th class="text-left">Soon <a href="https://github.com/WebAssembly/design/blob/master/PostMVP.md">after MVP</a></th>
+      <th class="text-left"><a href="https://github.com/WebAssembly/design/blob/master/FutureFeatures.md">Future versions</a></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>asm.js functional parity</td>
+      <td>threads</td>
+      <td>finer-grained memory control</td>
+    </tr>
+    <tr>
+      <td>validation semantics</td>
+      <td>shared memory</td>
+      <td>large page support</td>
+    </tr>
+    <tr>
+      <td>execution semantics</td>
+      <td>dynamic linking</td>
+      <td>more expressive control flow</td>
+    </tr>
+    <tr>
+      <td>module semantics</td>
+      <td>zero-cost exceptions</td>
+      <td>GC/DOM integration</td>
+    </tr>
+    <tr>
+      <td>binary encoding</td>
+      <td>fixed-width SIMD</td>
+      <td>linear memory >4GiB</td>
+    </tr>
+    <tr>
+      <td>textual format</td>
+      <td></td>
+      <td>... and <a href="https://github.com/WebAssembly/design/blob/master/FutureFeatures.md">more</td>
```

Comment:
looks like there was a bug in the original source here. can you close the link tag here?